### PR TITLE
[FIS2] Add formulas and Finish the UC

### PR DESCRIPTION
**Pull request by Pesteves2002**

### CODE REVIEW

**Comment by Pesteves2002** [comment]
@diogotcorreia @randomicecube what do u think?
@RafDevX maybe there are some typos 🧐

File: content/fis-ii/1000-formulario.md [hunk 1]
```diff
@@ -315,24 +323,86 @@ $K = 1 + \chi$
...
 $
 
-$\vec j$ - Densidade de Corrente $\sigma$ - Condutividade do Meio (Constante) $\vec f$ - força
+$\vec J$ - Densidade de Corrente $\ \ \sigma$ - Condutividade do Meio (Constante) $\vec f$ - força
 
 ### Lei de Ohm
+
+$R = \cfrac{V}I$
+
+$R$ - Resistência Elétrica $\ \ V$ - Difernça de Potencial $\ \ I$ - Corrente Elétrica
```
Comment (RafDevX) [inline content/fis-ii/1000-formulario.md:350]:
```suggestion
$R$ - Resistência Elétrica $\ \ V$ - Diferença de Potencial $\ \ I$ - Corrente Elétrica
```

File: content/fis-ii/1000-formulario.md [hunk 2]
```diff
@@ -315,24 +323,86 @@ $K = 1 + \chi$
...
+
+### Lei de Joule
+
+$P = VI = RI^2$
+
+$P$ - Potência Emitida (Dissipada) $\ \ V$ - Difernça de Potencial\
```
Comment (RafDevX) [inline content/fis-ii/1000-formulario.md:356]:
```suggestion
$P$ - Potência Emitida (Dissipada) $\ \ V$ - Diferença de Potencial\
```

File: content/fis-ii/1000-formulario.md [hunk 3]
```diff
@@ -315,24 +323,86 @@ $K = 1 + \chi$
...
+
+### Lei de Biot-Savart
+
+$
+\vec B (\vec r) = \cfrac {\mu_{0}}{4 \pi} \int_{C} \cfrac{\vec I \times \vec e_{r}}{r^2} \ dl  '
+$
+
+$vec B (\vec r)$ - Campo Magnético $\ \ \vec I$ - Corrente Elétrica $\ \ r$ - Distância a um Ponto
+
+### Lei de Ampére
```
Comment (RafDevX) [inline content/fis-ii/1000-formulario.md:374]:
```suggestion
### Lei de Ampère
```

**Review by RafDevX — CHANGES_REQUESTED** [review]
fix typo

File: content/fis-ii/1000-formulario.md (hunk 3 above)
Comment (diogotcorreia) [reply content/fis-ii/1000-formulario.md:374]: https://en.wikipedia.org/wiki/Andr%C3%A9-Marie_Amp%C3%A8re
No it isn't

**Review by diogotcorreia — CHANGES_REQUESTED** [review]
Resolve o typo do rafa, de resto está ready to merge

**Pull request merged**